EIPS/eip-1.md

```diff
@@ -290,6 +286,8 @@ Dec 4, 2019: The "Security Considerations" section was introduced.
 
 June 17, 2020: Canonicalizes the format for referencing EIPs by number in the "Style Guide".
 
+Sep 10, 2020: Require the `author` header to include at least one GitHub username.
+
```

Comment:
This also feels redundant with git commit history.  Why aren't we just letting our version control system do what version control systems do best (track change histories)?